lib/appwriteDB.ts

```
@@ -182,6 +202,201 @@ export class AppwriteDB {
     }
   }
 
+  // Get threads with pagination for current user
+  static async getThreadsPaginated(limit: number = 25, offset: number = 0): Promise<{
+    threads: Thread[];
+    hasMore: boolean;
+    total: number;
+  }> {
+    try {
+      const userId = await this.getCurrentUserId();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)